CRM/Volunteer/BAO/Need.php

```diff
@@ -58,6 +58,12 @@ function __construct() {
    * @static
    */
   static function &create($params) {
+    // these metadata fields are managed; don't accept them as params
+    unset($params['created'], $params['last_updated']);
```

Comment:
I suppose import is supported via Eileen's CSV to API tool.

I guess the use case you're suggesting is: I'm migrating from some other system, which keeps track of when opportunities were created/updated, and I don't want to lose that during the import.
I can maybe be talked into giving a developer enough rope to hang himself. If you want to continue this thread, probably better create a new issue because I'm sure it'll get lost here. Thanks!